README.md

```diff
@@ -163,6 +163,67 @@ IList<string> documentsToDelete = new List<string> {
 client.DocumentManager.AddOrUpdateDocuments(sourceId, documentsToAdd, documentsToDelete, null);
 ```
 
+## Using the SDK to push documents to the Stream API
+
+Methods to interact with the Stream API are part of the `client.DocumentManager` object. It’s a different mode of operations from the Push API, but very similar, with several more calls.
+
+### Requirements:
+
+* Before you proceed with this method, ensure you’ve understood how to [index data](https://docs.coveo.com/en/2956/coveo-for-commerce/index-commerce-catalog-content-with-the-stream-api) and [create a commerce catalog](https://docs.coveo.com/en/3139/coveo-for-commerce/create-a-coveo-commerce-catalog).
```

Comment:
I think the order is a bit weird, shouldn't the creation of the commerce catalog come first? I would also say 'index data with the Stream API'.

Reply:
> 

I apologize if this is causing extra work on your end. 
The next time around I'll be sure to ask for the proper access to the repo.